docs/network-policies.md

```diff
@@ -0,0 +1,90 @@
+# NetworkPolicy + OnlineBoutique
```

Comment:
**Suggestion:**
Let's move all these files (except, of course, `.github/release-cluster/README.md`) into a folder called `/docs/network-policies/`, and rename `rename-policies.md` to `README.md` so that we can have a structure similar to https://github.com/GoogleCloudPlatform/bank-of-anthos/tree/main/extras.
This means we'll move all the YAML files from this pull-request to `/docs/network-policies/` or `/docs/network-policies/manifests/`.
We would then need to update the `.md` files to make sure all commands/snippets are pointing to the correct folders.

Reply:
Gotcha, done.